cmd/zetae2e/local/local.go

```diff
@@ -265,14 +265,17 @@ func localE2ETest(cmd *cobra.Command, _ []string) {
 		}
 		zetaAdvancedTests := []string{
 			e2etests.TestZetaDepositRestrictedName,
+			e2etests.TestZetaDepositName,
```

Comment:
Is this a advanced test or because it's been already done at the very beginning?

Reply:
It is done by default to fund the account